package.json

```
@@ -74,6 +74,10 @@
       "import": "./dist/tokens/postcss/rem-to-px.js",
       "require": "./dist/tokens/postcss/rem-to-px.cjs"
     },
+    "./tokens/postcss/root-to-host": {
+      "import": "./dist/tokens/postcss/root-to-host.js",
+      "require": "./dist/tokens/postcss/root-to-host.cjs"
+    },
```

You can simplify this exports to:

```suggestion
    "./tokens/postcss/*": {
      "import": "./dist/tokens/postcss/*.js",
      "require": "./dist/tokens/postcss/*.cjs"
    },
```

This will include `rem-to-px` export too.